example/lib/main.dart

```diff
@@ -21,18 +21,30 @@ class MyApp extends StatelessWidget {
     return MaterialApp(
       title: 'FlutterBleLib example',
       theme: new ThemeData(
-        primaryColor: new Color(0xFF0A3D91),
-        accentColor: new Color(0xFFCC0000),
+        primaryColor: Color(0xFF0A3D91),
```

Comment:
I bet there is a file with defined colors

Reply:
Moved color definition to a dedicated file.